src/telemetry/telemetry.c

```diff
@@ -16,19 +25,59 @@
 
 #define MODULE_COUNT 2
 
+static downlink_node *downlink_queue = NULL;
```

Comment:
I think it would be better for this to be local in downlink_queue, and have the `read_downlink_queue` and `send_telemetry_local` not take the queue as an argument.

Reply:
It's the remnant of a previous version. Coming in the next commit.